VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/switch-on-camera-microphone/switch-on-camera-microphone.component.ts

```diff
@@ -146,12 +140,20 @@ export class SwitchOnCameraMicrophoneComponent extends ParticipantStatusBaseDire
         return this.conference.participants.some(x => x.user_name === this.profile.username && x.hearing_role === HearingRole.OBSERVER);
     }
 
-    private handleDevicesAvailable() {
+    private handleDevices(devicesAvailable: boolean): void {
```

Comment:
we might be able to fix the sonar issue here by changing the calling code:

`.subscribe(hasDevices => hasDevices ? this.handleDevicesAvailable() : this.handleDevicesUnavailable());`

then we shouldn't need this function anymore

Reply:
That's how the code was originally, I changed it to this to tackle the sonar issue, but it didn't make any difference. I can revert the change but it will still have that issue

Reply:
I see. I found this fixed the issue in admin web - might be worth a try? https://github.com/hmcts/vh-admin-web/pull/1442/files#diff-c451ff149851930d1a5b0d3676a579f57ba4894dae812458238de9044ea6e1dbR21

I think a combination of `subscribe({next` and use of the `?` check was enough to suppress the issue